costs/src/providers/openrouter/index.ts

```diff
@@ -7,587 +7,1513 @@ import { ModelRow } from "../../interfaces/Cost";
 
 export const costs: ModelRow[] = [
```

Comment:
In cost/ README.md, there are instructions on how to recreate snapshot tests and copy this to all of the other places it is needed.